Task5

```diff
@@ -0,0 +1,6 @@
+class Home
```

Comment:
This is wrong realization of task. The script should take the NAME OF TEXT FILE and the LINE.